app/edge/views/edge-stacks/createEdgeStackView/create-edge-stack-view.controller.js

```diff
@@ -165,14 +166,18 @@ export default class CreateEdgeStackViewController {
         this.state.Method = 'template';
         const template = await getCustomTemplate(templateId);
 
-        this.setTemplateValues({ template });
+        await this.setTemplateValues({ template }).then(() => {
+          this.state.loading = false;
+        });
```

Comment:
```suggestion
        await this.setTemplateValues({ template })
```

and then see my comment on the next code snippet